docs/custom-webview-android.md

```diff
@@ -177,6 +296,30 @@ public class CustomWebViewManager extends RNCWebViewManager {
 }
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+class CustomWebViewManager : RNCWebViewManager() {
+    override fun getExportedCustomDirectEventTypeConstants(): MutableMap<Any?, Any?>? {
+        val export =
+            if (super.getExportedCustomDirectEventTypeConstants() != null) {
+                super.getExportedCustomDirectEventTypeConstants()
+            } else {
+                MapBuilder.newHashMap<Any, Any?>()
+            }
```

Comment:
```suggestion
        val super = super.getExportedCustomDirectEventTypeConstants()
        val export = if (super != null) {
            super
        } else {
            MapBuilder.newHashMap<Any, Any?>()
        }
```

Reply:
super is a keyword, it cannot be used as a variable name

Reply:
Yeah my bad 🤦‍♂️ `superTypeConstants` also work

Reply:
Okay

Reply:
Done